internal/test/wmcb/wmcb_test.go

```diff
@@ -191,35 +182,36 @@ func (vm *wmcbVM) runTestConfigureCNI(t *testing.T) {
 // initializeTestBootstrapperFiles initializes the files required for initialize-kubelet
 func (vm *wmcbVM) initializeTestBootstrapperFiles() error {
 	// Create the temp directory
-	_, _, err := vm.Run(mkdirCmd(remoteDir), false)
+	_, err := vm.Run(mkdirCmd(remoteDir), false)
 	if err != nil {
 		return fmt.Errorf("unable to create remote directory %s: %v", remoteDir, err)
 	}
 
 	// Copy kubelet.exe to C:\Windows\Temp\
-	_, _, err = vm.Run("cp "+remoteDir+"\\kubelet.exe "+winTemp, true)
+	_, err = vm.Run("cp "+remoteDir+"kubelet.exe "+winTemp, true)
 	if err != nil {
-		return fmt.Errorf("unable to copy kubelet.exe to %s", winTemp)
+		return fmt.Errorf("unable to copy kubelet.exe to %s: %v", winTemp, err)
 	}
 
 	// Ignition v2.3.0 maps to Ignition config spec v3.1.0.
 	ignitionAcceptHeaderSpec := "application/vnd.coreos.ignition+json`;version=3.1.0"
 	// Download the worker ignition to C:\Windows\Tenp\ using the script that ignores the server cert
-	_, _, err = vm.Run(wgetIgnoreCertCmd+" -server https://api-int."+framework.ClusterAddress+":22623/config/worker"+
+	output, err := vm.Run(wgetIgnoreCertCmd+" -server https://"+framework.ClusterAddress+":22623/config/worker"+
 		" -output "+winTemp+"worker.ign"+" -acceptHeader "+ignitionAcceptHeaderSpec, true)
 	if err != nil {
-		return fmt.Errorf("unable to download worker.ign: %v", err)
+		return fmt.Errorf("unable to download worker.ign: %v : %s", err, output)
 	}
 
 	return nil
 }
 
 // remoteDownload downloads the tar file in url to the remoteDownloadFile location and checks if the SHA matches
 func (vm *wmcbVM) remoteDownload(pkg PkgInfo, remoteDownloadFile string) error {
-	_, stderr, err := vm.Run("if (!(Test-Path "+remoteDownloadFile+")) { wget "+pkg.getUrl()+" -o "+remoteDownloadFile+" }",
-		true)
+	useragent := "Wget/1.13.4"
+	cmd := wgetIgnoreCertCmd + " -server \"" + pkg.getUrl() + "\" -output \"" + remoteDownloadFile + "\" -useragent \"" + useragent + "\""
```

Comment:
the `-useragent` flag of `wgetIgnoreCertCmd` has been replaced with an `-acceptHeader` flag.

You can probably just pass it `*/*` as accept header here.

Reply:
thanks @LorbusChris  ! Noticed this after failed test. Changing it !